src/Error/Debug/dumpHeader.html

```diff
@@ -1,275 +1,284 @@
-<style type="text/css">
-.cake-debug {
-  --color-bg: #ECECE9;
-  --color-highlight-bg: #fcf8e3;
-  --color-control-bg: hsla(0, 0%, 50%, 0.2);
-
-  --color-orange: #c44f24;
-  --color-green: #0b6125;
-  --color-violet: #a71d5d;
-  --color-blue: #4070a0;
-  --color-grey: #5f5f5f;
-
-  --color-dark-grey: #222;
-  --color-cyan: #234aa0;
-  --color-red: #d33c44;
-
-  --indent: 20px;
-
-  font-family: monospace;
-  background: var(--color-bg);
-  padding: 5px;
-  line-height: 16px;
-  font-size: 14px;
-  margin-bottom: 10px;
-  position: relative;
-}
-.cake-debug:last-child {
-  margin-bottom: 0;
-}
-.cake-debug > span {
-  display: block;
-  margin-bottom: 10px;
-  color: var(--color-dark-grey);
-}
-
-.cake-dbg-object {
-  display: inline;
-}
-.cake-dbg-object[data-highlighted=true],
-.cake-dbg-object[data-highlighted=true] samp {
-  background: var(--color-highlight-bg);
-}
-
-/*
-Array item container and each items are blocks so
-nesting works.
-*/
-.cake-dbg-object-props,
-.cake-dbg-array-items {
-  display: block;
-}
-.cake-dbg-prop,
-.cake-dbg-array-item {
-  display: block;
-  padding-left: var(--indent);
-  min-height: 18px;
-}
-
-/** Collapser buttons **/
-[data-hidden=true] {
-  display: none;
-}
-
-.cake-dbg-collapse {
-  display: inline-block;
-  width: 14px;
-  height: 14px;
-  vertical-align: middle;
-  border-radius: 3px;
-  color: var(--color-blue);
-
-  background: var(--color-control-bg);
-  /* Image is an rawurlencoded SVG */
-  background-image: url("data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20height%3D%2212%22%20width%3D%2212%22%20viewBox%3D%220%200%2012%2012%22%3E%3Cpolygon%20points%3D%223%2C1%203%2C11%208%2C6%22 style%3D%22fill%3A%234070a0%3B%22%2F%3E%3C%2Fsvg%3E");
-  background-position: 2px 1px;
-  background-repeat: no-repeat;
-}
-.cake-dbg-collapse[data-open=true] {
-  transform: rotate(90deg);
-}
-/* Copy button */
-.cake-dbg-copy {
-  position: absolute;
-  top: 0px;
-  right: 0px;
-  padding: 6px;
-  background: var(--color-control-bg);
-  color: var(--color-blue);
-  border-radius: 0 0 0 3px;
-}
-
-/* Textual elements */
-.cake-dbg-class {
-  color: var(--color-cyan);
-}
-.cake-dbg-property {
-  color: var(--color-dark-grey);
-}
-.cake-dbg-visibility {
-  color: var(--color-violet);
-}
-.cake-dbg-punct {
-  color: var(--color-grey);
-}
-.cake-dbg-string {
-  color: var(--color-green);
-  white-space: pre-wrap;
-}
-.cake-dbg-number {
-  font-weight: bold;
-  color: var(--color-blue);
-}
-.cake-dbg-const {
-  color: var(--color-yellow);
```

Comment:
`--color-yellow` wasn't even defined but now we use the `--color-orange` (which had no usage) for constants and booleans.